src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/miner/IndustrialMiner.java

```diff
@@ -212,7 +212,7 @@ public void onInteract(Player p, Block b) {
      * @return Whether this {@link IndustrialMiner} is capable of mining this {@link Material}
      */
     public boolean canMine(Material type) {
-        return type.name().endsWith("_ORE") || (type == Material.ANCIENT_DEBRIS && canMineAncientDebris.getValue());
+        return type.name().endsWith("_ORE") || (SlimefunPlugin.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_16) && type == Material.ANCIENT_DEBRIS && canMineAncientDebris.getValue());
```

Comment:
Also, this is getting a bit too long for a single line I think.
It's probably better to restructure this:

if (ore) {
  return true;
}
else if (1.16) {
  return <debris stuff>
}
else {
  return false;
}

Reply:
Got it